src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +11,14 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
+
+        for (mate.academy.model.User user : users) {
```

Comment:
Let's import this class instead of using its full path here
```suggestion
        for (User user : users) {
```